scripts/build-ci.sh

```diff
@@ -1,66 +1,25 @@
 #!/bin/bash
-set -euo pipefail
-WORKSPACE="$(pwd)"
-KERNEL="$(uname -s)"
-MATCHINE="$(uname -m)"
-CURL_VERSION="8.6.0_1"
-WOA_TOOLCHAIN_VERSION="2024-02-08"
-MINGW_CURL_WIN64_BLOB="https://curl.se/windows/dl-$CURL_VERSION/curl-$CURL_VERSION-win64-mingw.zip"
-MINGW_CURL_WIN64A_BLOB="https://curl.se/windows/dl-$CURL_VERSION/curl-$CURL_VERSION-win64a-mingw.zip"
-MINGW32_TOOLCHAIN_BLOB="https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/releases/download/$WOA_TOOLCHAIN_VERSION/aarch64-w64-mingw32-msvcrt-toolchain.tar.gz"
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
-case "$KERNEL" in
-Linux)
-    KERNEL="linux"
-    ;;
-Darwin)
-    KERNEL="darwin"
-    MATCHINE="universal"
-    ;;
-esac
-
-function download {
-    URL="$1"
-    SAVED_PATH="$(mktemp)"
-    SAVED_DIR="$(mktemp -d)"
-    wget --no-verbose "$URL" -O "$SAVED_PATH"
-    case "$URL" in
-    *.zip)
-        unzip -q -d "$SAVED_DIR" "$SAVED_PATH"
-        rm "$SAVED_PATH"
-        echo "$SAVED_DIR"
-        ;;
-    *.tar.gz)
-        tar -C "$SAVED_DIR" --gzip --extract --file="$SAVED_PATH"
-        rm "$SAVED_PATH"
-        echo "$SAVED_DIR"
-        ;;
-    *)
-        echo "$SAVED_PATH"
-        ;;
-    esac
-}
+source "$SCRIPT_DIR/functions.sh"
 
-set -x
+set -xeuo pipefail
```

Comment:
`set -x` will pollute output, use it only when debugging.

Reply:
i want the user to know which commands were run

Reply:
The user should read the script, instead of DoS the brain of maintainers.

Reply:
OK